tests/type_inference/test_literals.py

```diff
@@ -27,6 +27,23 @@ def test_simple_literal(const):
     assert [type(const)] == result
 
 
+# what do we want to give as input? Randomly generated tuples of varying sizes.
+# @given
+# def test_tuple():
```

Comment:
@RyanDJLee please don't comment out code like this in general, and certainly do not do it in a commit. You should be reviewing all changes you made before committing your work.